site/en/docs/extensions/reference/storage/index.md

```diff
@@ -107,14 +154,14 @@ chrome.storage.onChanged.addListener(function (changes, namespace) {
 });
 ```
 
-We can take this idea even further. In this example we have an [options page][options-page] that
-allows the user to toggle a "debug mode" in the extension (implementation not shown here). Changes
-to this setting are immediately saved to sync storage by the options page and the background script uses `storage.onChanged` to apply the setting as soon as possible.
+We can take this idea even further. In this example, we have an [options page][options-page] that
+allows the user to toggle a "debug mode" (implementation not shown here).  The options page immediately saves the new settings to `storage.sync`, and the service worker uses `storage.onChanged` to apply the setting as soon as possible.
 
 {% Label %}options.html:{% endLabel %}
 
 ```html
-<script defer src="options.js"></script>
+<!-- type="module" allows you to use top level await -->
+<script defer src="options.js" type="module"></script>
 <form id="optionsForm">
```

Comment:
Non-blocking FYI: HTML attribute values are not case sensitive. As such, you will typically HTML attribute values use snake-case while JavaScript variables are typically camelCase.